File: Asocioacion/php/tercerEjercicio/Aeronave.php
<?php

class Aeronave{

    private string $numeroMatricula;
    private string $nombreOperador;
    private string $fabricanteNave;
    private int $modeloNave;

    public function __construct(string $numeroMatricula, string $nombreOperador, string $fabricanteNave, int $modeloNave){

        $this->numeroMatricula = $numeroMatricula;
        $this->nombreOperador = $nombreOperador;
        $this->fabricanteNave = $fabricanteNave;
        $this->modeloNave = $modeloNave;

    }

    //Numero de la matricula
    public function getNumeroMatricula(){
        return $this->numeroMatricula;
    }
    public function setNumeroMatricula($numeroMatricula){
        $this->numeroMatricula = $numeroMatricula;
    }

    //Nombre del operador
    public function getNombreOperador(){
        return $this->nombreOperador;
    }
    public function setNombreOperador($nombreOperador){
        $this->nombreOperador = $nombreOperador;
    }

    //Fabricante de la nave
    public function getFabricanteNave(){
        return $this->fabricanteNave;
    }
    public function setFabricanteNave($fabricanteNave){
        $this->fabricanteNave = $fabricanteNave;
    }

    //Modelo de la nave

    public function getModeloNave(){
        return $this->modeloNave;
    }
    public function setModeloNave($modeloNave){
        $this->modeloNave = $modeloNave;
    }
}

?>

File: Asocioacion/python/tercerEjercicio/Aeronave.py
class Aeronave:
    
    c = 0
    
    def __init__(self, numMatricula, nombreOperador, fabricanteNave, modeloNave):
        self.numMatricula = numMatricula
        self.nombreOperador = nombreOperador
        self.fabricanteNave = fabricanteNave
        self.modeloNave = modeloNave
        self.__class__.c+=1
        
    #Numero de matricula
    
    def getNumMatricula(self):
        return self.numMatricula
    
    def setNumMatricula(self,numMatricula):
        self.numMatricula = numMatricula
        
    #Nombre de operador
    
    def getNombreOperador(self):
        return self.nombreOperador
    
    def setNombreOperador(self, nombreOperador):
        self.nombreOperador = nombreOperador
        
    #Fabricante de la nave
    
    def getFabricanteNave(self):
        return self.fabricanteNave
    
    def setFabricanteNave(self,fabricanteNave):
        self.fabricanteNave = fabricanteNave
        
    #Modelo de la nave
    
    def getModeloNave(self):
        return self.modeloNave
    
    def setModeloNave(self, modeloNave):
        self.modeloNave = modeloNave
        